src/edu/csus/ecs/pc2/core/imports/clics/FileComparison.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.report.ComparisonState;
+import edu.csus.ecs.pc2.core.report.FileComparisonUtilities;
+
+/**
+ * Comparison information between two files.
+ * 
+ * @author Douglas A. Lane, PC^2 team pc2@ecs.csus.edu
+ */
+public class FileComparison {
+
+    @JsonProperty
+    private String firstFilename;
+
+    @JsonProperty
+    private String secondFilename;
+
+    @JsonProperty
+    long numberRows;
+    
+    @JsonProperty
+    long numberDifferences;
+
+    @JsonProperty
+    long numberComparedRecorderds;
```

Comment:
This variable name doesn't make sense.  Perhaps it was intended to be `numberComparedRecords` ?

Reply:
typo, fixed